contrib/test/integration/test.yml

```diff
@@ -24,9 +27,10 @@
       when: not integration_selinux_enabled
 
     - name: run integration tests
-      shell: "CGROUP_MANAGER=cgroupfs STORAGE_OPTIONS='--storage-driver=overlay{{ extra_storage_opts | default('') }}' make localintegration >& {{ artifacts }}/testout.txt"
+      shell: "{{ int_test_cmd }}"
       args:
         chdir: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-incubator/cri-o"
+      environment: '{{ int_test_env }}'  # from vars.yml
```

Comment:
Misleading comment, because `int_test_env` gets redefined just a few lines above.

Reply:
True, I guess it assumes people read from top to bottom :smile: Should I just remove it then, or do you have another idea?